Fix errors in set-manifest.md (#76)
edgelesssys/marblerun.sh: content/docs/workflows/set-manifest.md

It further defines what Infrastructure providers are allowed.
You can set a manifest through Marblerun's Client HTTP REST API.
The endpoint for all manifest operations is `/manifest`.
See [Defining a manifest]({{< ref "docs/workflows/define-manifest.md" >}}) to learn how to define a manifest.

> To set the manifest we can use the command line interface, which performs remote attestation before uploading the manifest.
> Before the manifest is uploaded, the coordinator is verified.
> For further information see [Verifying a deployment]({{< ref "docs/workflows/verification.md" >}})

Reviewer: ```suggestion
To set the manifest we can use the command line interface, which performs remote attestation on the Coordinator before uploading the manifest.
For further information see [Verifying a deployment]({{< ref "docs/workflows/verification.md" >}})
```
No need to tell people twice about the remote attestation
